### ISSUE TITLE
Add a true_axes_names argument to the dump() method? (xlwings)

### ISSUE THREAD

**Issue by alixdamman** [larray-project/larray#820]
The ``dump()`` of the Array class uses the ``AxisCollection.display_names`` property to generate the axes names. This means that exporting arrays with anonymous axes will convert the ``None`` names as strings ``{i}*``.

However, I am not sure it's worth it to add a new argument to the ``dump()`` method.

@gdementen 
What do you think?

**Comment by gdementen** [comment]
Honestly, I fail to see a case where a user would want {i}* instead of nothing at all (an empty string!) in an Excel file.

Do you think using .names in .dump instead of .display_names would break anything? If not, I think that's better than adding yet another option that nobody will ever use.

**Comment by gdementen** [comment]
My former self probably found it elegant to have Excel tables, CSV files and the string representation of arrays as similar as possible, but my current self disagrees 😉.

**Comment by alixdamman** [comment]
Well, ``dump()`` is used by ``Array.__str__()``. So we really need an extra argument but we can choose to define `axes_names` as `self.axes.names` by default and `self.axes.display_names` otherwise.

**Comment by gdementen** [comment]
Yes, indeed. Maybe a private argument name (`_axes_display_names=False`)?

**Comment by alixdamman** [comment]
Good idea.
